modules/05_initialCap/on/preloop.gms

```diff
@@ -532,12 +532,24 @@ if (cm_startyear gt 2005,
   Execute_Loadpoint 'input_ref' pm_EN_demand_from_initialcap2 = pm_EN_demand_from_initialcap2;
   Execute_Loadpoint 'input_ref' pm_pedem_res = pm_pedem_res;
   Execute_Loadpoint 'input_ref' pm_dataeta = pm_dataeta;
-  Execute_Loadpoint 'input_ref' pm_data = pm_data;
   Execute_Loadpoint 'input_ref' pm_aux_capLowerLimit = pm_aux_capLowerLimit;
   Execute_Loadpoint 'input_ref' vm_deltaCap.l = vm_deltaCap.l;
   Execute_Loadpoint 'input_ref' vm_deltaCap.lo = vm_deltaCap.lo;
   Execute_Loadpoint 'input_ref' vm_deltaCap.up = vm_deltaCap.up;
 
+
+
+*** load pm_data from input_ref.gdx and overwrite values
+*** only for eta of chp technologies since they have been adapted in initialCap routine above
+*** This is to avoid overwriting changes to pm_data by scenario switches
+  Execute_Loadpoint 'input_ref' p05_pmdata_ref = pm_data;
+  pm_data(regi,char,te)$( (sameas(te,"coalchp")  
```

Comment:
I have to admit I don't get your fix. 
If I understand correctly, it only changes pm_data. 
If I am not mistaken, pm_data(eta) is not used in the model anymore, but only the two parameters pm_dataeta and pm_eta_conv, which are still overwriten just like before ?

Reply:
ok, maybe my understanding was wrong - you simply left pm_data(eta) to be also overwritten, as this was changed, but made sure that pm_data itself is not overwritten.
sorry for my slow thinking :-)

Reply:
Thx, all good :)

I did not know that the "eta" values in ``pm_data`` are not used anymore in the model afterwards. Maybe we should collect such items for the technology cost refactoring. Can we note this somewhere, @cchrisgong?